tests/platforms/kubernetes/app_description.go

```diff
@@ -16,6 +16,7 @@ type AppDescription struct {
 	RegistryName      string
 	Replicas          int32
 	IngressEnabled    bool
+	EnableMetrics     string
```

Comment:
nit: `MetricsEnabled` to follow the pattern from similar properties above.

Reply:
@artursouza I actually had it called that however the setting is `--enabled-metrics` throughout the Dapr components so I stuck with `EnabledMetrics` for consistency.

Reply:
I understand that but you can see how this struct follows a different pattern.

Reply:
I changed it back and also made it a `bool` to be consistent with the other settings. That means having to set it to `true` because that is was the behavior before. It's likely the e2e tests don't require metrics I but played it safe and set it to `true` everywhere.